src/theme/theme.js

```diff
@@ -12,6 +12,16 @@ export const baseTheme = EditorView.theme({
   '&.cm-editor.cm-focused': {
     outline: 'none',
   },
+  '& .cm-completionInfo': {
```

Comment:
As indicated in the weekly: Is there a chance for us to provide examples as _verbatim_ (monospace)? I think that would benefit the eye.

If not, then this looks just as we need it!

Reply:
I would handle mono-space in the example extractor. In the feel-editor, this would affect all variable info and I don't think we want this

Reply:
:+1: 

I'm not 100% sure how we can best support that. You're the expert.

At some point we'd want to improve our previews anyway. We could event consider to not act until that date.

Reply:
> I'm not 100% sure how we can best support that. You're the expert.

Codemirror supports passing DOM elements. I did not have a deeper look, so this might blow up when we merge multiple elements currently, but thats the direction I would want to go

Reply:
:+1: